crates/drainer/src/main.rs

```diff
@@ -20,6 +20,9 @@ async fn main() -> DrainerResult<()> {
     let shutdown_intervals = conf.drainer.shutdown_interval;
     let loop_interval = conf.drainer.loop_interval;
 
+    #[cfg(feature = "vergen")]
+    println!("Starting drainer (Version: {})", router_env::git_tag!());
```

Comment:
```suggestion
    eprintln!("Starting drainer (Version: {})", router_env::git_tag!());
```
should we print to stderr?
or maybe print this in json compatible format maybe...

Reply:
I wouldn't prefer printing to stderr, as this isn't exactly some information about something going wrong.

As for the JSON format, I don't think it's absolutely necessary, since we're also logging out the console and file logging filters directly to stdout. If there's a reason to use JSON other than this being parsed by Loki/Promtail, please do let me know.